brainscore_vision/model_helpers/activations/core.py

```diff
@@ -33,16 +36,41 @@ def __init__(self, get_activations, preprocessing, identifier=False, batch_size=
         self._stimulus_set_hooks = {}
         self._batch_activations_hooks = {}
 
-    def __call__(self, stimuli, layers, stimuli_identifier=None):
+    def __call__(self, stimuli, layers, stimuli_identifier=None, model_requirements=None):
```

Comment:
I don't like that this exposes the micro-saccades to the benchmark. It's a modeling choice of if/how to do micro-saccades. E.g. models with micro-saccades would yield different responses over multiple trials of the same stimulus (we already have `number_of_trials`).